### PR TITLE
Accessibility: Improve navigation and contact approval

### PR DESCRIPTION
**Pull request by annando**
And the next step. The navigation bar now works better with screen readers. (There is still an issue with the notifications though, that I cannot fix right now)

Also the contact request now works well with screenreaders. And we used some ARIA attributes in a wrong way.

### CODE REVIEW

File: view/templates/notifications/intros.tpl [hunk 1]
```diff
@@ -20,12 +20,16 @@
 
 <form class="intro-approve-form" action="{{$action}}" method="post">
 {{include file="field_checkbox.tpl" field=$hidden}}
+<div role="radiogroup" aria-labelledby="connection_type">
+	<div id="connection_type">{{$lbl_connection_type}}</div>
```
Comment (MrPetovan) [inline view/templates/notifications/intros.tpl]: Looks like it should be a `<label>`.

File: view/theme/frio/templates/nav.tpl [hunk 2]
```diff
@@ -38,41 +38,40 @@
 
 				{{* Left section of the NavBar with navigation shortcuts/icons *}}
 				<ul class="nav navbar-nav navbar-left" role="menubar">
-					<li id="nav-communication" class="nav-segment" role="presentation">
+					<li id="nav-communication" class="nav-segment">
 						{{if $nav.network}}
-						<a accesskey="n" role="menuitem" class="nav-menu {{$sel.network}}" href="{{$nav.network.0}}" data-toggle="tooltip" aria-label="{{$nav.network.3}}" title="{{$nav.network.3}}"><i class="fa fa-lg fa-th" aria-hidden="true"></i><span id="net-update" class="nav-network-badge badge nav-notification"></span></a>
```
Comment (MrPetovan) [inline view/theme/frio/templates/nav.tpl]: Why the removal of these attributes? I believe they are recommended by Bootstrap.

File: view/theme/frio/templates/notifications/intros.tpl [hunk 3]
```diff
@@ -47,21 +38,34 @@
...
+
 		{{* This sections contains special settings for contact approval. We hide it by default and load this section in
 		a bootstrap modal in the case of approval *}}
 		<div id="intro-approve-wrapper-{{$intro_id}}" style="display: none;">
 
 			<h3 class="heading">{{$fullname}}{{if $addr}}&nbsp;({{$addr}}){{/if}}</h3>
 			<form class="intro-approve-form" {{if $request}}action="{{$request}}" method="get"{{else}}action="{{$action}}" method="post"{{/if}}>
 				{{include file="field_checkbox.tpl" field=$hidden}}
+				<div role="radiogroup" aria-labelledby="connection_type">
+					<div id="connection_type">{{$lbl_connection_type}}</div>
```
Comment (MrPetovan) [inline view/theme/frio/templates/notifications/intros.tpl]: `<label>`

File: view/theme/frio/templates/nav.tpl (hunk 2 above)
Comment (annando) [reply view/theme/frio/templates/nav.tpl]: You mean this `role="menuitem"`? Well, I was told so. See also this article: https://marcozehe.de/2019/05/30/wai-aria-menus-use-with-care/

**Comment by MrPetovan** [comment]
Have you checked the display is the same between block `<div>` and inline `<label>`?

**Comment by annando** [comment]
Well, the labels are printed in bold, but that shouldn't be a problem.

**Pull request merged**